molecule/default/converge.yml

```
@@ -653,6 +661,8 @@
                       core:
                         index: frontend_index.html
                         root: /usr/share/nginx/html
+                      http2:
+                        chunk_size: 8k
```

Lets test a different value to ensure the defaults are being correctly replaced
```suggestion
                        chunk_size: 10k
```